nixos/modules/services/audio/spotifyd.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.spotifyd;
+  spotifydConf = pkgs.writeText "spotifyd.conf" cfg.config;
+  spotifyd = pkgs.callPackage (import ../../../../pkgs/applications/audio/spotifyd/default.nix) { };
```

Comment:
```suggestion
```

Can be accessed from `pkgs.spotifyd`

Reply:
Oops, you're right -- I left that in from when I was testing something locally.